added initial search result example (#75)
MrLongbottom/P9-Rapport: sections/appendix/search_result.tex

\subsection{Search Example}
In this section, the best model configuration for document queries, \texttt{\gls{bm25}} * \texttt{\gls{pr}} is tested against a query to see what our search engine.

\subsection{Document query}
The query we test is:
\begin{quotation}
	\textit{eriksen landshold hareide holdkammerat}
\end{quotation}
This query is generally about football and should therefore return similar articles regarding the same topic.
\textit{Eriksen} is referring to a Danish football player(Christian Eriksen) and \textit{Hareide} is referring to the Danish national football team's coach.

\noindent Headline of the top 5 article is shown below.
\begin{enumerate}

> 	\item Danmarks største stjerne er i storform

Reviewer: ```suggestion
	\item \textbf{Danmarks største stjerne er i storform}
```

	\item Profils formkurve er opadgående
	\item Åge Hareide modtager pris
	\item Fodboldlandshold rykker frem

> 	\item Grå irere er vant til playoff

Reviewer: can we textbf the right one if its in there

\subsection{Search Example}

> In this section, the best model configuration for document queries, \texttt{\gls{bm25}} * \texttt{\gls{pr}} is tested against a query to see what our search engine.

Reviewer 2: ```suggestion
In this section, the best model configuration for document queries, \texttt{\gls{bm25}} * \texttt{\gls{pr}} is tested against a query to see what our search engine retrieves.
```

> \subsection{Document query}

Reviewer 2: ```suggestion
\subsection{Document Query}
```

The query we test is:
\begin{quotation}
	\textit{eriksen landshold hareide holdkammerat}
\end{quotation}
This query is generally about football and should therefore return similar articles regarding the same topic.
\textit{Eriksen} is referring to a Danish football player(Christian Eriksen) and \textit{Hareide} is referring to the Danish national football team's coach.

> \noindent Headline of the top 5 article is shown below.

Reviewer 2: ```suggestion
\noindent Headlines of the top 5 articles are shown below.
```

\begin{enumerate}
	\item \textbf{Danmarks største stjerne er i storform}
	\item Profils formkurve er opadgående
	\item Åge Hareide modtager pris
	\item Fodboldlandshold rykker frem
	\item Grå irere er vant til playoff
\end{enumerate}

> The headline in bold is the articles that we were searching for with the given query. 

Reviewer 2: ```suggestion
The headline in bold is the article that we were searching for with the given query. 
```

These articles are generally concerned with the topic of football or giving awards to players/coaches which indicates good retrieval performance.

> \subsection{Topic query}

Reviewer 2: ```suggestion
\subsection{Topic Query}
```

The query we test is:
\begin{quotation}
	\textit{dilemma plejehjem nordthy betjene}
\end{quotation}

> The query is about a dilemma at a nursing home in nordthy where some police officer were involved. 

Reviewer 2: ```suggestion
The query would seem to be about a dilemma at a nursing home in Nordthy where some police officers were involved. 
```

This might not make sense, since the words are possibly taken from different articles, which might introduce more noise.

> \noindent Headline of the top 5 article is shown below.

Reviewer 2: ```suggestion
\noindent Headlines of the top 5 articles are shown below.
```

\begin{enumerate}
	\item Nordthy pløjer sig gennem puljen
	\item Nordthy skarpest
	\item Nordthy vandt knebent
	\item Brændte chancer kostede
	\item Ikke skarpe nok
\end{enumerate}

> All of the articles, which are found in top 5 from the query are about either football or handball where Nordthy is mentioned.

Reviewer 2: ```suggestion
All of the articles, which are found in top 5 from the query, are about either football or handball where Nordthy is mentioned.
```

> This indicates that the \gls{bm25} has too much influence on which articles are selected based on the query.

Reviewer 2: ```suggestion
This indicates that the \gls{bm25} method has too much influence on which articles are selected based on the query.
```
